client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -158,6 +160,10 @@ class CalypsoifyIframe extends Component< Props & ConnectedProps & ProtectedForm
 
 			// Check if we're generating a post via Press This
 			this.pressThis();
+
+			// Notify any external listeners that the iframe has loaded
+			Dispatcher.handleViewAction( this.props.editorLoaded() );
```

Comment:
You don't need to import and use the Flux dispatcher here. Simply calling
```js
this.props.editorLoaded();
```
will dispatch the action to Redux.

Reply:
Nice. Removed in 24ba01d.